package.json

```diff
@@ -66,6 +66,7 @@
     "mocha": "^8.3.1",
     "mocha-lcov-reporter": "^1.3.0",
     "pouchdb-adapter-memory": "^7.2.2",
+    "pouchdb-debug": "^7.2.1",
```

Comment:
Is this used?

Reply:
I think we should keep it here so I can be used when needed. It was used to see what the find code was doing but I took that logging out. We can either keep that logging in or remove this if necessary. The debugger is a helpful tool though.

Reply:
If we're going to keep the the library we need to at least have commented logging code or else anyone else auditing deps will just ask this question again in the future (or remove it).